medmodels/statistic_evaluations/statistical_analysis/inferential_statistics.pyi

```diff
@@ -0,0 +1,19 @@
+from typing import Dict, List, Union
+
+from numpy.typing import ArrayLike
+
+class TestHypothesis:
```

Comment:
For all the functions within this class. How is it that none of them take any arguments. How will they be used?

Also, I would change the naming
```suggestion
class StatisticalTest:
```